modules/gapi/test/common/gapi_imgproc_tests_inl.hpp

```diff
@@ -50,6 +50,27 @@ namespace
             rgb2yuyv(in_line_p, out_line_p, in.cols);
         }
     }
+
+    // Draw random ellipses on given mat of given size and type
+    void initMatForFindingContours(cv::Mat& mat, const cv::Size& sz, int type)
```

Comment:
const int type

Reply:
Agree, added `const `  in `checkMeta` in `imgproc.hpp` too